eclair-core/src/main/scala/fr/acinq/eclair/payment/send/PaymentInitiator.scala

```diff
@@ -192,7 +192,7 @@ object PaymentInitiator {
    *                           the payment will automatically be retried in case of TrampolineFeeInsufficient errors.
    *                           For example, [(10 msat, 144), (15 msat, 288)] will first send a payment with a fee of 10
    *                           msat and cltv of 144, and retry with 15 msat and 288 in case an error occurs.
-   * @param finalExpiryDelta   expiry delta for the final recipient.
+   * @param finalExpiryDelta   expiry delta for the final recipient when the [[paymentRequest]] doesn't specify it.
```

Comment:
The name is confusing, it doesn't show that this is a fallback. Maybe `defaultfinalExpiryDelta`?

Reply:
Yeah I agree, why not even `fallbackFinalExpiryDelta`?

Reply:
Done in https://github.com/ACINQ/eclair/pull/1483/commits/d28fbba9ef0c69ec911610b2f381297fd62e32a3